packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -40,83 +40,38 @@ function BlockStylesPanel( { clientId } ) {
 	);
 }
 
-function BlockInspectorLockedBlocks( { topLevelLockedBlock } ) {
```

Comment:
Instead of creating a custom "inspector control" for "content only" mode (and similar), I refactored this component to reuse the same block inspector control component by passing a "isContentLockingParent" flag instead.

Reply:
This makes everything easier to reason about 👍